doc/BW64.rst

```diff
@@ -0,0 +1,113 @@
+BW64 I/O
+========
+
+To read or write a BW64 file, the primary interface is the
+:func:`openBw64Adm` and :func:`openBw64` functions.
+
+To read parsed ADM metadata and samples from a file, use something like:
```

Comment:
This suggests that the ADM within the file has to be in a format that is (somehow) already parsed.  I think the meaning is
"To read samples, and to read and parse ADM metadata, from a file, use something like:"

Reply:
Good point, should be fixed (and generally a bit better).